### ISSUE TITLE
GlobeSurfaceTileProvider test failures

### ISSUE THREAD

**Issue by tfili** [AnalyticalGraphicsInc/cesium#3978]
I got the following test failures in master. Using `Chrome Version 51.0.2704.63 m (64-bit)`. This doesn't happen in 1.21.

```
Scene/GlobeSurfaceTileProvider renders in 2D geographic
Expected [ 0, 0, 0, 255 ] not to equal [ 0, 0, 0, 255 ].
Error: Expected [ 0, 0, 0, 255 ] not to equal [ 0, 0, 0, 255 ].
    at stack (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1475:17)
    at buildExpectationResult (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1445:14)
    at Spec.expectationResultFactory (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:584:18)
    at Spec.addExpectationResult (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:332:34)
    at Expectation.addExpectationResult (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:528:21)
    at Expectation.toEqual (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1399:12)
    at http://localhost:8080/Specs/Scene/GlobeSurfaceTileProviderSpec.js:300:48
    at Object.then (http://localhost:8080/Source/ThirdParty/when.js:196:34)
    at http://localhost:8080/Source/ThirdParty/when.js:297:13
    at processQueue (http://localhost:8080/Source/ThirdParty/when.js:647:4)
Scene/GlobeSurfaceTileProvider renders in 2D web mercator
Expected [ 0, 0, 0, 255 ] not to equal [ 0, 0, 0, 255 ].
Error: Expected [ 0, 0, 0, 255 ] not to equal [ 0, 0, 0, 255 ].
    at stack (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1475:17)
    at buildExpectationResult (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1445:14)
    at Spec.expectationResultFactory (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:584:18)
    at Spec.addExpectationResult (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:332:34)
    at Expectation.addExpectationResult (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:528:21)
    at Expectation.toEqual (http://localhost:8080/ThirdParty/jasmine-2.2.0/jasmine.js:1399:12)
    at http://localhost:8080/Specs/Scene/GlobeSurfaceTileProviderSpec.js:314:48
    at Object.then (http://localhost:8080/Source/ThirdParty/when.js:196:34)
    at http://localhost:8080/Source/ThirdParty/when.js:297:13
    at processQueue (http://localhost:8080/Source/ThirdParty/when.js:647:4)
```

**Comment by pjcozzi** [comment]
These test failures were not in Chrome 50 on Mac, but I see them now in Chrome 51 on Mac and Windows, and Firefox 46.

@bagnell @lilleyse can one of you look at this?

**Comment by tfili** [comment]
Fixed via #3985.
